tango_ros_common/tango_ros_messages/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(tango_ros_messages)
+
+find_package(catkin REQUIRED COMPONENTS
+   message_generation
+   std_msgs
+   )
+
+add_service_files(
+  DIRECTORY srv
+  FILES
+  SaveMap.srv
+)
+
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+)
+
+catkin_package(
+  INCLUDE_DIRS include
```

Comment:
When compiling this locally in a clean workspace I got an error that no include dir exists in package. So the line above should probably be removed.